docs/docs/glossary.md

```diff
@@ -37,7 +41,7 @@ How this works will be discussed via a future RFP process on Discourse, similarl
 
 ### Private Execution Environment
 
-The Private eXecution Environment (PXE) is a client-side library for the execution of private operations. The PXE generates proofs of private function execution, and sends these proofs along with public function requests to the sequencer. Private inputs never leave the client-side PXE.
+<Glossary.PXE />
 
 Read more [here](./aztec/concepts/pxe/index.md).
 
```

Comment:
Further down, add a line for Testing Execution Environment (TXE)

Reply:
Also one for "Sandbox"

Reply:
Done